packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -311,7 +306,7 @@ describe('KeyringController', () => {
         await withController(async ({ controller, initialState }) => {
           const [primaryKeyring] = controller.getKeyringsByType(
             KeyringTypes.hd,
-          ) as Keyring<Json>[];
+          ) as EthKeyring[];
```

Comment:
We probably want to use `Keyring` here (from `@metamask/keyring-utils`) since `EthKeyring` is a "special keyring type" that also supports user ops methods.

This comment is also true for all occurences to `EthKeyring[]` (the ones that got changed on this PR at least)

Reply:
In this case, the method being called is `addNewAccountForKeyring` which expects an `EthKeyring`?

Reply:
My bad, you are right, I didn't realized we were already using this type everywhere in the `KeyringController`! Sorry 🙈